api/src/pcapi/core/subscription/commands.py

```diff
@@ -13,3 +18,35 @@
 @log_cron_with_transaction
 def send_beneficiary_subscription_reminders() -> None:
     reminder_emails.send_reminder_emails()
+
+
+@blueprint.cli.command("send_reminders_to_users_that_landed_on_come_back_later")
+@log_cron_with_transaction
+def send_reminders_to_users_that_landed_on_come_back_later() -> None:
```

Comment:
Nom plus court à trouver. Quand tu vois le mot "that" dans une variable c'est mauvais signe.
users_that_landed_on_come_back_later -> stand_by_users ?

Reply:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** `‏send_come_back_later_users_reminder` ?